libs/utils/utils.js

```diff
@@ -233,6 +233,9 @@ export function createTag(tag, attributes, html) {
       el.setAttribute(key, val);
     });
   }
+  if (options.parent) {
+    options.parent.append(el);
```

Comment:
Would it be worth checking that `options.parent` is an element before appending stuff to it?

Reply:
i don't think so, if someone wants to use something else as a parent, so be it...the day we have a documentation, we'll make it clear.
This said i'll add a dedicated unit test

Reply:
> the day we have a documentation, we'll make it clear

this is the number 1 thing lacking from not having a build step for milo